cmd/queue/main.go

```diff
@@ -463,3 +470,13 @@ func flush(logger *zap.SugaredLogger) {
 	os.Stderr.Sync()
 	metrics.FlushExporter()
 }
+
+func enableConcurrencyState(logger *zap.SugaredLogger, env config) bool {
```

Comment:
nit: this reads to me like it will enable concurrency state, rather than checking whether concurrency state is enabled. How about something like `concurrencyStateEnabled`?

We could also do what `tracingEnabled` does and drop the method in favour of `concurrencyStateHookEnabled := env.ConcurrencyStateEndpoint != ""`

Reply:
Probably better to follow the convention, so will drop the method